### ISSUE TITLE
`tilt@0.12.0` breaks `rules_docker` support.

### ISSUE THREAD

**Issue by wtgtybhertgeghgtwtg** [windmilleng/tilt#2928]
Using the `bazel_example` as an example, `tilt@0.12.0` will try to find `bazel_vigoda:image`, while `go_image` will push `bazel/vigoda:image`.

**Comment by nicks** [comment]
thanks for the report! This sounds pretty bad, let me poke around at this.

**Comment by nicks** [comment]
Hi @wtgtybhertgeghgtwtg ! I'm having trouble reproducing this. 

Are you just running `tilt up` in the bazel_example repo?

Can you send me the results of your `tilt doctor` and/or a snapshot of what you're seeing?

**Comment by nicks** [comment]
(for what it's worth, I'm seeing a issue with microk8s where it's not properly pushing to the local registry at localhost:32000. But it's not clear to me that's a new issue, or if it's the same issue as what you're seeing. Other clusters seem to work ok.)

**Comment by wtgtybhertgeghgtwtg** [comment]
Sorry about that.  Here's the output of `tilt doctor`
```
Tilt: v0.12.0, built 2020-02-07
System: linux-amd64
---
Docker
- Host: [default]
- Version: 1.40
- Builder: 2
---
Kubernetes
- Env: kind-0.6+
- Context: kind-kind
- Cluster Name: kind-kind
- Namespace: default
- Container Runtime: containerd
- Version: v1.17.0
---
Thanks for seeing the Tilt Doctor!
Please send the info above when filing bug reports. 💗

The info below helps us understand how you're using Tilt so we can improve,
but is not required to ask for help.
---
Analytics Settings
- User Mode: opt-in
- Machine: fd69a37997fd560aa2117c9f9640557d
- Repo: UR3R9lDkfThx22nlHUqubw==
```
And here's the snapshot: https://cloud.tilt.dev/snapshot/AcrRluQLG0KyrxLo3yw=

**Comment by nicks** [comment]
Thanks! I see the problem. The local kind registry stuff you're using isn't playing nicely with the custom_build stuff. Lemme see if we can do a quick fix.

**Comment by nicks** [comment]
The fix for this is in v0.12.1, which we released this morning. Thanks again for reporting!
